### PR TITLE
Improve system and sensor scheduling

### PR DESCRIPTION
**Pull request by LorenzMeier**
This change will improve the timing accuracy by putting the high priority work queue into the correct scheduling category.

### CODE REVIEW

File: nuttx-configs/px4fmu-v1/nsh/defconfig [hunk 1]
```diff
@@ -728,7 +728,7 @@ CONFIG_MQ_MAXMSGSIZE=32
 #
 CONFIG_SCHED_WORKQUEUE=y
 CONFIG_SCHED_HPWORK=y
-CONFIG_SCHED_HPWORKPRIORITY=192
+CONFIG_SCHED_HPWORKPRIORITY=249
```
Comment (dagar) [inline nuttx-configs/px4fmu-v1/nsh/defconfig]: Drop FMUv1 to avoid rebasing? https://github.com/PX4/Firmware/pull/7769

**Comment by davids5** [comment]
@LorenzMeier - I think this make good sense.  

image


As @dagar - I would omit V1.

Would you like me to instrument SPI and I2C buses and quantify the jitter?

File: nuttx-configs/px4fmu-v1/nsh/defconfig (hunk 1 above)
Comment (LorenzMeier) [inline nuttx-configs/px4fmu-v1/nsh/defconfig]: Good point, I dropped the commit.

**Comment by LorenzMeier** [comment]
Thanks! I brought that all in.

**Pull request merged**